### DISCUSSION TITLE
Female header sockets with the Teensy?

### DISCUSSION THREAD

**Discussion by parmour** [FundamentalFrequency/.github#82] [General]
Hi,

sorry if this is a trivial question but when building my LMN-3 I'm inclined to use female header sockets rather than soldering directly to the board as the build guide says.
Am I right to assume there is no problem with that?

Thanks in advance!

**Comment by stonepreston** [comment]
yeah thats fine many people do that to make reusing the teensy easier, just make sure whatever headers you choose are short enough so that when the teensy is mounted it still fits into the case

**Reply by parmour** [reply]
Thanks, yes I went ahead and it works fine.
